### PR TITLE
Bug/fix markup validation issues

### PR DESCRIPTION
**Pull request by joshfeck**
## Problem this Pull Request solves

Lots of html validation errors in the help screens. Mostly the issue was unordered lists (`<ul>`) were wrapped in paragraph tags (`<p>`). In addition to the invalid markup, there were a few places where there was an opening tag with now closing tag, and a few places with a closing tag with no opening tag.

## How has this been tested

Click the help tab on just about every admin page and view the various screens. They should all have valid markup and display as expected. 

## Checklist

* [x] I have read the documentation relating to systems affected by this pull request, see https://github.com/eventespresso/event-espresso-core/tree/master/docs
* [x] User input is adequately validated and sanitized
* [x] all publicly displayed strings are internationalized (usually using `esc_html__()`, see https://codex.wordpress.org/I18n_for_WordPress_Developers)
* [x] My code is tested.
* [x] My code follows the Event Espresso code style.
* [x] My code has proper inline documentation.
* [x] My code accounts for when the site is in Maintenance Mode (MM2 especially disallows usage of models)

### CODE REVIEW

File: admin_pages/registration_form/help_tabs/registration_form_questions_overview_table_column_headings.help_tab.php [hunk 1]
```diff
@@ -1,59 +1,57 @@
-<p><strong><?php _e('Questions Overview Table Column Headings', 'event_espresso'); ?></strong></p>
-<p>
+<p><strong><?php esc_html_e('Questions Overview Table Column Headings', 'event_espresso'); ?></strong></p>
 <ul>
 <li>
-<strong><?php _e('ID', 'event_espresso'); ?></strong><br />
-<?php _e('This is the numerical ID for the question. This value is used internally for Event Espresso.', 'event_espresso'); ?>
+<strong><?php esc_html_e('ID', 'event_espresso'); ?></strong><br />
+<?php esc_html_e('This is the numerical ID for the question. This value is used internally for Event Espresso.', 'event_espresso'); ?>
```
Comment (nerrad) [inline admin_pages/registration_form/help_tabs/registration_form_questions_overview_table_column_headings.help_tab.php]: Should we use the new brand name helper for the `Event Espresso` brand string here and elsewhere in this set of changes?

**Comment by nerrad** [comment]
I'll leave my approval, but there are a number of linting issues introduced (mostly because spaces are being used for indenting instead of tabs for our php files.

**Comment by joshfeck** [comment]
I went through and added the branding helper method, it makes sense to start using it now that it's been merged. Can you review the latest?

**Comment by sethshoultes** [comment]
I reviewed all of the help tabs throughout the EE admin and don't see any HTML formatting issues. Thanks for tackling this Josh!

**Pull request merged**